3460/0/triangle.rb

```diff
@@ -0,0 +1,37 @@
+
+# frozen_string_literal: true
+
+# Triangle Project Code.
+
+# Triangle analyzes the lengths of the sides of a triangle
+# (represented by a, b and c) and returns the type of triangle.
+#
+# It returns:
+#   :equilateral  if all sides are equal
+#   :isosceles    if exactly 2 sides are equal
+#   :scalene      if no sides are equal
+#
+# The tests for this method can be found in
+#   about_triangle_project.rb
+# and
+#   about_triangle_project_2.rb
+#
+
+def triangle_kind(side_a, side_b, side_c)
+  return :equilateral if (side_a == side_b) && (side_b == side_c)
+  return :isosceles if (side_a == side_b) || (side_b == side_c) || (side_c == side_a)
```

Comment:
LGTM.
You can also try to use `[side_a, side_b, side_c].uniq.size` as `case` argument. Or even replace case with hash.

Reply:
I use .min .max